Source/Classes/AnimatedImages/PINCachedAnimatedImage.m

```diff
@@ -97,9 +98,6 @@ - (instancetype)initWithAnimatedImage:(id <PINAnimatedImage>)animatedImage
         // dispatch later so that blocks can be set after init this runloop
         dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
             [self imageAtIndex:0];
```

Comment:
@garrettmoon shouldn't this line call -coverImage, which seems like it will load image at index 0 if needed, but also will trigger the coverImageReady callback like it did before this change?